setup.py

```diff
@@ -274,8 +274,9 @@ def run(self):
       platforms=['any'],
       packages=get_packages(),
       package_data={LIBNAME: get_package_data(LIBNAME, EXTLIST),
-                    'spyderplugins':
-                    get_package_data('spyderplugins', EXTLIST)},
+                    'spyplugins': get_package_data('spyplugins', EXTLIST),
+                    },
+      include_package_data=True,
```

Comment:
Ok, please see if it's needed or not by running:

    python setup.py bdist_wheel --universal

and checking if `spyplugins` is part of the wheel or not.